WolvenKit.App/ViewModels/Documents/MeshPreviewBlock.cs

```diff
@@ -322,6 +322,12 @@ public void OnMouseDown3DHandler(object sender, MouseDown3DEventArgs e)
                     SelectedItem = group;
                     e.Handled = true;
                 }
+                if (e.HitTestResult.ModelHit is SubmeshComponent lod)
```

Comment:
Could use pattern matching for that
`e.HitTestResult.ModelHit is SubmeshComponent { Parent: MeshComponent { Parent: MeshComponent mesh } }`

Reply:
Good to know, added that. Thanks a lot :D